src/neo/VM/Helper.cs

```diff
@@ -298,6 +299,8 @@ public static StackItem ToStackItem(this ContractParameter parameter)
 
         private static StackItem ToStackItem(ContractParameter parameter, List<(StackItem, ContractParameter)> context)
         {
+            if (parameter is null) throw new ArgumentNullException();
+            if (parameter.Value is null) return StackItem.Null;
```

Comment:
The problem of move this check here is that `InteropInterface` will returns `Null` instead of an  `Exception`

Reply:
Ok the previous test was with null value....